### PR TITLE
Create an article

### PR DESCRIPTION
**Pull request by Jlo6CTEP**

### CODE REVIEW

File: docs/en/anticorona_drones.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+### Introduction
+* The world faces the worst pandemic of XXI century, which affects lifes and well-being of millions of citisens. To slow down the spread of the disease and give the healthcare system to react, people are obliged to wear masks, however, people sometimes ignore those rules, which puts lifes of others under the threat. 
+There are laws in place to enforce mask wearing, but there is just not enough cops to monitor the situation in every mall, bus station etc and fine the law-breakers. We want to contribute to the solution of this problem, but instead of punishing people, we want to provide them with mask with our super-friendly and cute drone.
+
+### Custom Airframe
+* Since we do not have the Clover drone, we will use an airframe built and designed by us.
+
+* The main idea is to use truss structure, because it works well against twisting and warping 
+![](https://i.imgur.com/NpX6YnA.png)
```
Comment (okalachev) [inline docs/en/anticorona_drones.md:22]: Please, don't use external images, commit them to the repo instead.

**Review by okalachev — CHANGES_REQUESTED** [review]

**Comment by okalachev** [comment]
Please, review my editing, and say, is it okay.

File: docs/en/anticorona_drones.md (hunk 1 above)
Comment (Jlo6CTEP) [reply docs/en/anticorona_drones.md]: Done!

**Comment by Jlo6CTEP** [comment]
Requested changes integrated

**Comment by okalachev** [comment]
Give me some your contact (e-mail/Telegram) to add it to the article.

**Pull request merged**